storops/unity/enums.py

```diff
@@ -394,6 +394,7 @@ class HostTypeEnum(UnityEnum):
     NET_GROUP = (3, 'Net Group')
     RPA = (4, 'RecoverPoint Appliance')
     HOST_AUTO = (5, 'Auto-managed Host')
+    VNXSanCopy = (255, 'VNX Block Migration system')
```

Comment:
Do we need to check other newly added enum values as Harrier gets released?

Reply:
Reasonable, will did a check then.

BTW, this enum was added since 4.1 from the api changes.

Reply:
@amagicsun, All updated enums since Unity 4.1 are included in this patch

Reply:
Great, thanks!